test/controllers/forms/EducationQuestionnaireFormExamples.scala

```diff
@@ -52,7 +52,7 @@ object EducationQuestionnaireFormExamples {
     None,
     Some(true))
 
-  val NoUkLivedAndNoCivilServantValidForm = EducationQuestionnaireForm.Data(
+  val NoUkLivedAndNoDegreeValidForm = EducationQuestionnaireForm.Data(
```

Comment:
Shall we change NoUkLivedAndNoDegreeValidForm -> NotLivedInUkAndNoDegreeValidForm pls